pennylane_qiskit/converter.py

```diff
@@ -584,14 +585,17 @@ def _function(*args, params: dict = None, wires: list = None, **kwargs):
     return _function
 
 
-def load_qasm(qasm_string: str):
+def load_qasm(qasm_string: str, measurements=None):
```

Comment:
This is more of a discussion point, but I wonder if the default value here should be `[]` instead of `None`? 

That would preserve the current behaviour if someone is using this without passing measurements. But then this should also carry through to the PennyLane PR, and it will mean that the behaviour between `from_qiskit` and `from_qasm` is inconsistent without measurements.

I would be in favour of updating to this implementation, but then I guess this should be included in the changelog as a breaking change. For context, [here is a previous PR](https://github.com/PennyLaneAI/pennylane-qiskit/pull/469) where we specifically defaulted to `[]` when calling `load` (below) to _avoid_ making this breaking change. But I think if we add the `measurements` UI that `from_qiskit` has to this function, then its time to make the breaking change. @trbromley @obliviateandsurrender

Reply:
See comment in the test file below for a concrete example of the behaviour that will change with this update

Reply:
@MashAliK , we've discussed this offline and we'll stick with the implementation you have here for the PR, no need to update anything